tests/multisigNFTs_test.go

```diff
@@ -79,136 +69,16 @@ func makeMultisigTxsNFTs() []*testCase {
-
-			//Accept-token-ownership : multiSig-delete
-			{"multiSig", true, true, "Delete MultiSig Tx for NFTs [accept-token-ownership] -  Error, due to Group address invalid.                         ", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"multiSig-delete-tx", "multisig-acc-1", "", 0, nil, "grp-addr-4", 0, internalTx3_acceptTokenOwnership_00}, "MEMO : Delete MultiSig Tx", nil},
-			///--- start : due to above [//------------------------------- start : Transfer-ownership of Token]
-			{"multiSig", true, true, "Delete MultiSig Tx for NFTs [accept-token-ownership] -  Error, Invalid Owner address due to Only group account owner can remove pending tx.                         ", "multisig-acc-4", "100000000cin", 0, MultisigInfo{"multiSig-delete-tx", "multisig-acc-4", "", 0, nil, "grp-addr-2", 0, internalTx3_acceptTokenOwnership_00}, "MEMO : Delete MultiSig Tx", nil},
-			{"multiSig", true, true, "Delete MultiSig Tx for NFTs [accept-token-ownership] -  Error, due to Pending tx is not found which ID : 42.      ", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"multiSig-delete-tx", "multisig-acc-1", "", 0, nil, "grp-addr-2", 42, internalTx3_acceptTokenOwnership_00}, "MEMO : Delete MultiSig Tx", nil},
-
-			//Accept-token-ownership : multiSig-transfer-ownership
-			{"multiSig", true, true, "Transfer MultiSig Owner for NFTs [accept-token-ownership] -  Error, due to Group address invalid.                                                                ", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-1", "multisig-acc-2", 0, nil, "grp-addr-4", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil}, //
-			{"multiSig", true, true, "Transfer MultiSig Owner for NFTs [accept-token-ownership] -  Error, due to Owner of group address invalid.                                                       ", "multisig-acc-3", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-3", "multisig-acc-1", 0, nil, "grp-addr-2", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil}, //
-			{"multiSig", true, true, "Transfer MultiSig Owner for NFTs [accept-token-ownership] -  Error, due to without KYC																																					 ", "multisig-acc-no-kyc", "100000000cin", 0, MultisigInfo{"transfer-ownership", "multisig-acc-1", "multisig-acc-no-kyc", 0, nil, "grp-addr-2", 0, nil}, "MEMO : Transfer MultiSig Owner.", nil}, //
-
-		*/
+		// case-1 :  with ONE signer, should broadcast immediately
+		{"multiSig", false, false, "Create MultiSig Tx for NFTs [Create-token] - submit counter+0 - Happy Path commit.", "multisig-acc-1", "100000000cin", 0, MultisigInfo{"create-internal-tx", "", "", 0, nil, "grp-addr-1", 0, internalTx4}, "MEMO : xxxxx", nil},
```

Comment:
Title : 
NFTs-multisig - Issues of Create-multisig-Tx & Sign-multisig-Tx

Hi, Mostafa

I able to create those tx and sign base on them (refer case-1, case-2), with these below category :
1. With ONE signer, should broadcast immediately (line-73)
2. With TWO signers  (line-76 to 77)


However, I have questions below :

1. Although successfully committed into the blockchain, but still can see the pendingTx which should be null after done the Sign-multisig-Tx. Please refer samples as attached.

2. How do I proceed for the Approve-token using the Token-owner, as u drafted.    

3. I already included this below in file 'tx_test.go' :
-- strings.Contains(tc.desc, "wait-3-seconds")


Please comment. TQ. 


![mostafa-NFTs](https://user-images.githubusercontent.com/35713528/78326595-62775080-75ad-11ea-8f9b-602eb76f103c.png)
[samples_with ONE signer_(CreateTx-SignTx).txt](https://github.com/maxonrow/maxonrow-go/files/4425154/samples_with.ONE.signer_.CreateTx-SignTx.txt)

[samples_with TWO signers_(CreateTx-SignTx).txt](https://github.com/maxonrow/maxonrow-go/files/4425155/samples_with.TWO.signers_.CreateTx-SignTx.txt)

Reply:
Hi, Mostafa

I able to find the bugs why can not be broadcasted immediately once reach the threshold, which was due to :  
-- Broadcasting internal transaction failed with error message of 'Fee collector invalid.'


I able to create those tx and sign base on them which resulting PendingTx = NULL, with these below category.

1. With ONE signer, should broadcast immediately : 
Refer - samples_with ONE signer_(CreateTx-SignTx).txt

2. With TWO signers :
Refer - samples_with TWO signers_(CreateTx-SignTx).txt



However, I have questions below :

1. How do I proceed for the Approve-token using the Token-owner, as u drafted.

Please comment. TQ.

[samples_with ONE signer_(CreateTx-SignTx).txt](https://github.com/maxonrow/maxonrow-go/files/4436165/samples_with.ONE.signer_.CreateTx-SignTx.txt)

[samples_with TWO signers_(CreateTx-SignTx).txt](https://github.com/maxonrow/maxonrow-go/files/4436167/samples_with.TWO.signers_.CreateTx-SignTx.txt)

Reply:
After 3 or 4 seconds transaction should be broadcasted. then you can approve the token

Reply:
Hi, Mostafa

I will follow up on this 'Approve Token' and update.

Reply:
Hi, Mostafa

During the Approve-nft-token, this function 'makeApproveNonFungibleTokenMsg'
need parameters of Provider, Issuer & endorserList.

What address I need to use for above value ? Is that Group-address or owner-Group-address ?

Please comment.

Reply:
Dear all,

Discussed done, Goh already follow-up this.

Reply:
Dear all,

I close this as done, pls refer to this latest :

https://github.com/maxonrow/maxonrow-go/pull/97